### DISCUSSION TITLE
Modifying fonts of code blocks independently from inline code

### DISCUSSION THREAD

**Discussion by iincer** [squidfunk/mkdocs-material#5342] [Q&A] [answered]
First, thank you for developing this excellent tool!
I am working on a site in which we display a fair amount of code. We have both code blocks and inline code. Is there a good way to modify the font size of code blocks independently from the size of inline code? By a code block, I mean blocks we enclose with \`\`\`, and by inline code, statements enclosed by \`.

We are currently modifying `.md-typeset code`, but this affects both inline code and code blocks.

**Comment by squidfunk** [comment, marked as answer]
Try this:

``` css
.md-typeset pre {
  font-size: 18px;
}
```

**Comment by iincer** [comment]
Thanks so much. This took care of it.
